### ISSUE TITLE
Issues with Iris Shaders for Fabric. No block textures!

### ISSUE THREAD

**Issue by sho310gh** [maruohon/litematica#928]
Since updating to Minecraft 1.21.4 and all the mods as well, I experience an issue with litematica. Blocks are not rendered with texture anymore. Some items like the pot or banner still appear. Here are the versions I currently use:

- FABRIC 0.16.9
- Fabric API 0.113.0
- Litematica Fabric 0.21.0
- MaLiLib Fabric 0.23.0
- Sodium Fabric 0.6.6
- Iris Shaders Fabric 1.8.5
- Complementary Shaders - Reimagined r5.3

I've found some issues while using this mods/shaders and with other shaders too

- Hologram Blocks don't appear
- Borders appear, but opacity filter doesn't work (full transparent or full opaque)
- Somehow, blocks are not rendered, but shadows do

![2024-12-28_14 06 03](https://github.com/user-attachments/assets/19762f61-4520-4712-b8f4-05eaf47c7397)

**Comment by CMMJ1** [comment]
Your shaders could be the reason for that, you should try turning off shaders.

**Comment by sho310gh** [comment]
Yes, when I disable the shader it shows the texture, but is this not still an issue on litematica's side? This shows that it is not compatible with shaders or am I wrong and it's actually a problem on the shaders side?

**Comment by MCLady** [comment]
Did you ever find a fix for this, i am also having the same issue, tried a variety of shaders and none seem to be compatible

**Comment by maruohon** [comment]
The latest 1.21.4 mod release 0.21.2 has a fix for shader/Iris compatibility: https://modrinth.com/mod/litematica/version/0.21.2
